.claude/hooks/utils/llm/ollama_provider.py

```
@@ -0,0 +1,193 @@
+#!/usr/bin/env -S uv run --script
```

Ruff (0.12.2)

Shebang is present but file is not executable

(EXE001)